### ISSUE TITLE
"Delete"-Commands in "Navigating the Command Line" do not actually (only) delete

### ISSUE THREAD

**Issue by NicApicella** [dwmkerr/effective-shell#48]
In the "Navigating the Command Line" section, there are a bunch of commands described as "deleting" things. Also in [website/content/docs/section1/1-navigating-the-command-line/images/command-line.png the image] commands are subdivided into "Move" and "Delete".

**The thing is, readline does not actually _delete_!** More accurately, does "not only" delete: it rather _kills_ (comparable to an—enhanced—cut command).

The short version is that, as a result of killing instead of merely deleting, you can later _yank_ the killed text back (comparable to paste). (The longer version is that there exists a _kill ring_ and that opens up a whole other can of worms. :-) But it's awesome! And worth mentioning! At the very least, "delete" is simply … too simplistic. ;-) )

**Comment by dwmkerr** [comment]
Hi @NicApicella that's awesome info, thanks for sharing! I'll update the chapter with more data on this 😄  can't wait to dig into the kill ring for my own curiosity BTW!

**Comment by dwmkerr** [comment]
Backfilling the manuscript does not fix this, therefore keeping open, however will be version 1.X post book launch
